cypress-tests/cypress.config.js

```diff
@@ -15,11 +25,45 @@ module.exports = {
           return null;
         },
       });
+      on("after:screenshot", (details) => {
+        // Full path to the screenshot file
+        const screenshotPath = details.path;
+
+        // Extract filename without extension
+        const name = path.basename(
+          screenshotPath,
+          path.extname(screenshotPath)
+        );
+
+        // Define a new name with a connectorId
+        const newName = `[${connectorId}] ${name}.png`;
+        const newPath = path.join(path.dirname(screenshotPath), newName);
```

Comment:
File name would be: `[<connector_name>] spec_file -- top_level_describe -- test_name (failed)`